### PR TITLE
i18n

### PR DESCRIPTION
**Pull request by mishuagopian**

### CODE REVIEW

**Comment by mishuagopian** [comment]
@epintos @sbalay

File: src/app/app.translations.js [hunk 1]
```diff
@@ -3,15 +3,21 @@ app.config([
...
-      SUNDAY: 'Domingo',
-      MONDAY: 'Lunes',
-      TUESDAY: 'Martes',
-      WEDNESDAY: 'Miércoles',
-      THURSDAY: 'Jueves',
-      FRIDAY: 'Viernes',
-      SATURDAY: 'Sabado'
-    }
+    SIGN_UP_FORM: {
+      SIGN_UP: 'Registrarme',
```
Comment (epintos) [inline src/app/app.translations.js]: Its better to order keys alphabeticallly. I know its boring, but it helps finding keys later.

**Comment by epintos** [comment]
Little fix and :+1:

**Comment by epintos** [comment]
Remember not to ammend commits when making pull requests fixes. You can make new commits and then squash at the end

:+1:

**Pull request merged**